### ISSUE TITLE
Esquisse functionality and unwanted filters

### ISSUE THREAD

**Issue by lcy22** [dreamRs/esquisse#100]
Hello, 

I am trying out the esquisse package for the first time and am having issues. I have tried running it in a browser and dialog box, but neither are working correctly. I also tried running in inside of a shiny dashboard but am getting the same issues. 

Mine does not look as nice as what I am seeing in tutorial videos online. I can not scroll and get stuck on the data page when trying to load anything to graph. I can drag and drop once I get one data frame in there, but nothing appears. When I click on the code it shows that all of the values seem to be filtered out. 

I tried running it in R and Rstudio as well as updating all my packages. I don't know what else could be causing this. 

Thanks in advance for the help! 
![Capture](https://user-images.githubusercontent.com/60787801/74043551-34033d80-498f-11ea-8153-27b3506846a1.PNG)
![Capture2](https://user-images.githubusercontent.com/60787801/74043806-abd16800-498f-11ea-84b9-982be9916c03.PNG)

**Comment by pvictor** [comment]
Hello,
Yes something is definitely not right... Have you tried to close RStudio and re-try ?
Can you provide result of `sessioninfo::session_info()` and `rstudioapi::versionInfo()$version` please ?

Victor

**Comment by lcy22** [comment]
Yes, I have tried closing and re-trying as well as clearing my global environment completely. Here is the information you requested. Thank you in advance for your help. 

- Session info -------------------------------------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.6.1 (2019-07-05)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/Chicago             
 date     2020-02-12                  

- Packages -----------------------------------------------------------------------------------------------------------------------------------------------
 package        * version date       lib source        
 assertthat       0.2.1   2019-03-21 [1] CRAN (R 3.6.1)
 cli              2.0.1   2020-01-08 [1] CRAN (R 3.6.2)
 colorspace       1.4-1   2019-03-18 [1] CRAN (R 3.6.1)
 crayon           1.3.4   2017-09-16 [1] CRAN (R 3.6.1)
 digest           0.6.23  2019-11-23 [1] CRAN (R 3.6.2)
 dplyr          * 0.8.4   2020-01-31 [1] CRAN (R 3.6.2)
 esquisse       * 0.3.0   2020-01-27 [1] CRAN (R 3.6.2)
 evaluate         0.14    2019-05-28 [1] CRAN (R 3.6.1)
 extrafont        0.17    2014-12-08 [1] CRAN (R 3.6.0)
 extrafontdb      1.0     2012-06-11 [1] CRAN (R 3.6.0)
 fansi            0.4.1   2020-01-08 [1] CRAN (R 3.6.2)
 farver           2.0.3   2020-01-16 [1] CRAN (R 3.6.2)
 fastmap          1.0.1   2019-10-08 [1] CRAN (R 3.6.1)
 gdtools          0.2.1   2019-10-14 [1] CRAN (R 3.6.2)
 ggplot2        * 3.2.1   2019-08-10 [1] CRAN (R 3.6.1)
 glue             1.3.1   2019-03-12 [1] CRAN (R 3.6.1)
 gtable           0.3.0   2019-03-25 [1] CRAN (R 3.6.1)
 hrbrthemes       0.6.0   2019-01-21 [1] CRAN (R 3.6.1)
 htmltools        0.4.0   2019-10-04 [1] CRAN (R 3.6.1)
 httpuv           1.5.2   2019-09-11 [1] CRAN (R 3.6.1)
 jsonlite         1.6.1   2020-02-02 [1] CRAN (R 3.6.2)
 knitr            1.25    2019-09-18 [1] CRAN (R 3.6.1)
 later            1.0.0   2019-10-04 [1] CRAN (R 3.6.1)
 lazyeval         0.2.2   2019-03-15 [1] CRAN (R 3.6.1)
 lifecycle        0.1.0   2019-08-01 [1] CRAN (R 3.6.1)
 magrittr         1.5     2014-11-22 [1] CRAN (R 3.6.1)
 mime             0.8     2019-12-19 [1] CRAN (R 3.6.2)
 miniUI           0.1.1.1 2018-05-18 [1] CRAN (R 3.6.1)
 munsell          0.5.0   2018-06-12 [1] CRAN (R 3.6.1)
 pillar           1.4.3   2019-12-20 [1] CRAN (R 3.6.2)
 pkgconfig        2.0.3   2019-09-22 [1] CRAN (R 3.6.1)
 promises         1.1.0   2019-10-04 [1] CRAN (R 3.6.1)
 purrr            0.3.3   2019-10-18 [1] CRAN (R 3.6.2)
 R6               2.4.1   2019-11-12 [1] CRAN (R 3.6.2)
 RColorBrewer     1.1-2   2014-12-07 [1] CRAN (R 3.6.0)
 Rcpp             1.0.3   2019-11-08 [1] CRAN (R 3.6.2)
 rlang            0.4.4   2020-01-28 [1] CRAN (R 3.6.2)
 rmarkdown        2.1     2020-01-20 [1] CRAN (R 3.6.2)
 rstudioapi       0.10    2019-03-19 [1] CRAN (R 3.6.1)
 Rttf2pt1         1.3.8   2020-01-10 [1] CRAN (R 3.6.2)
 scales           1.1.0   2019-11-18 [1] CRAN (R 3.6.2)
 sessioninfo      1.1.1   2018-11-05 [1] CRAN (R 3.6.1)
 shiny          * 1.4.0   2019-10-10 [1] CRAN (R 3.6.1)
 shinydashboard * 0.7.1   2018-10-17 [1] CRAN (R 3.6.2)
 shinyWidgets     0.5.0   2019-11-18 [1] CRAN (R 3.6.2)
 stringi          1.4.4   2020-01-09 [1] CRAN (R 3.6.2)
 systemfonts      0.1.1   2019-07-01 [1] CRAN (R 3.6.1)
 tibble           2.1.3   2019-06-06 [1] CRAN (R 3.6.1)
 tidyselect       1.0.0   2020-01-27 [1] CRAN (R 3.6.2)
 viridisLite      0.3.0   2018-02-01 [1] CRAN (R 3.6.1)
 withr            2.1.2   2018-03-15 [1] CRAN (R 3.6.1)
 xfun             0.12    2020-01-13 [1] CRAN (R 3.6.2)
 xtable           1.8-4   2019-04-21 [1] CRAN (R 3.6.1)

[1] \\cp6dfs/user/LCYeas/R/win-library/3.6
[2] C:/Program Files/R/R-3.6.1/library

Version:  ‘1.2.5001’

**Comment by pvictor** [comment]
Thks for the info., What's the code you used to launch esquisse exactly ?
